public/index.html

```diff
@@ -24,6 +24,9 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
+
```

Comment:
You're quite right - I must have overlooked that when I checked (obviously not thoroughly enough! I'll amend it now.